src/main/java/stsjorbsmod/cards/wanderer/ForbiddenGrimoire.java

```diff
@@ -27,11 +27,12 @@
     private static final int COST = 0;
     private static final int CARD_PLAYS_TO_EXHUME = 3;
 
+    public static final int EXHUME_TURN = 7;
+
     public ForbiddenGrimoire() {
         super(ID, COST, TYPE, COLOR, RARITY, TARGET);
         magicNumber = baseMagicNumber = CARD_PLAYS_TO_EXHUME;
         EntombedField.entombed.set(this, true);
-        SelfExhumeFields.selfExhumeAtStartOfTurn7.set(this, true);
```

Comment:
Was removing this intentional? It doesn't look like the new marker power has this effect

Reply:
Probably one or two too many ctrl-z's 🤦‍♂